src/Microsoft.Diagnostics.Runtime/src/DataTargets/IDacLocator.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Runtime
+{
+    public interface IDacLocator
```

Comment:
Do we really need this?  The idea behind the CreateRuntime method is that there's an overload for you to provide a path to the dac.  If you want to bypass all dac location in ClrMD, just find the dac yourself before calling CreateRuntime, then pass that local path to it.

Reply:
Also, should "category" be a string or a well defined enum?  If it's an enum users can filter on standardized categories.